src/renderer/components/+cluster-settings/components/cluster-prometheus-setting.tsx

```diff
@@ -26,18 +73,32 @@ export class ClusterPrometheusSetting extends React.Component<Props> {
           <a href="https://github.com/lensapp/lens/blob/master/troubleshooting/custom-prometheus.md" target="_blank">guide</a>{" "}
           for possible configuration changes.
         </p>
+        <p>Prometheus installation method.</p>
         <Select
-          value={this.props.cluster.preferences.prometheusProvider?.type || ""}
+          value={this.provider}
           onChange={({value}) => {
-            const provider = {
-              prometheusProvider: {
-                type: value
-              }
-            }
-            merge(this.props.cluster.preferences, provider);
+            this.provider = value;
+            this.onSaveProvider();
           }}
           options={options}
         />
+        <span className="hint">What query format is used to fetch metrics from Prometheus</span>
+        {this.canEditPrometheusPath && (
+          <>
+            <p>Prometheus service address.</p>
+            <Input
+              theme="round-black"
+              value={this.path}
+              onChange={(value) => this.path = value}
+              onBlur={this.onSavePath}
+              placeholder="<namespace>/<service>:<port>"
+            />
+            <span className="hint">
+              An address to an existing Prometheus installation{" "}
+              (&lt;namespace&gt;/&lt;service&gt;:&lt;port&gt;). Lens tries to auto-detect address if left empty.
```

Comment:
Better to avoid manual encoding html-entities and delegate this to React with:
`{'<namespace>/<service>:<port>'}`
Also this improves reading/updating texts.